xcube/core/store/accessors/dataset.py

```diff
@@ -52,20 +54,60 @@ def write_data(self, data: xr.Dataset, data_id: str, replace=False, **write_para
         data.to_netcdf(data_id, **write_params)
 
 
-class DatasetZarrPosixAccessor(PosixDataDeleter, DataWriter, DataOpener):
-    """
-    Extension name: "dataset:zarr:posix"
-    """
+class ZarrOpenerParamsSchemaMixin:
 
+    # noinspection PyUnusedLocal,PyMethodMayBeStatic
     def get_open_data_params_schema(self, data_id: str = None) -> JsonObjectSchema:
-        # TODO: implement me
-        return JsonObjectSchema()
+        return JsonObjectSchema(
+            properties=dict(
+                group=JsonStringSchema(
+                    description='Group path. (a.k.a. path in zarr terminology.).',
+                    min_length=1,
+                ),
+                chunks=JsonObjectSchema(
+                    description='Optional chunk sizes along each dimension. ',
+                    examples=[{'time': None, 'lat': 'auto', 'lon': 90},
+                              {'time': 1, 'y': 512, 'x': 512}],
+                    additional_properties=True,
+                ),
+                decode_cf=JsonBooleanSchema(
+                    description='Whether to decode these variables, assuming they were saved '
+                                'according to CF conventions.',
+                    default=True,
+                ),
+                mask_and_scale=JsonBooleanSchema(
+                    description='If True, replace array values equal to attribute "_FillValue" with NaN. '
+                                'Use attribute "scaling_factor" and "add_offset" to computed actual values.',
```

Comment:
What happens otherwise?

Reply:
Just negate what is said there. But that's really redundant:

'If False, do nit replace array values equal to attribute "_FillValue" with NaN. '
                                'Do not use attributes "scaling_factor" and "add_offset" to compute actual values.'

Reply:
Fixed some typos in description.